templates/image-builder.yml

```diff
@@ -34,6 +34,64 @@ objects:
     selector:
       matchLabels:
         name: image-builder
+    strategy:
+      type: Rolling
+      rollingParams:
+        pre:
```

Comment:
@major so k8s' Deployment doesn't have the pre-start lifecycle hook available under strategy, only ocp's DeploymentConfig does. Would you have any objections to switch to a DeploymentConfig?